docs/remme-framework.rst

```diff
@@ -0,0 +1,146 @@
+REMME Framework
+=================
+
+*************
+Serialisation
```

Comment:
Just wondering, doesn't 'Serialization' the correct word for that ?

Reply:
We make a full orthography review, thank you!

Reply:
Both are correct, one is UK spelling and the other is US spelling :)